### DISCUSSION TITLE
How to use Python-like f-strings

### DISCUSSION THREAD

**Discussion by gautam-e** [modular/modular#238] [Q&A] [answered]
I tried the following :
```
x = 'Mojo'
print(f"Hello {x}!")
```
But got this error:
```
error: Expression [13]:18:12: expected ')' in call argument list
    print(f"Hello {x}!")
```

How can I get a simple thing like this running in Mojo? I am using the Mojo Playground.

Obviously it works when I use `%%python` but I want to get it working in Mojo.

**Comment by mfranzon** [comment, marked as answer]
I think it is on the roadmap #86

**Reply by gautam-e** [reply]
Thanks! Will follow that thread.

**Comment by gautam-e** [comment]
I figured out how to do something similar and wrote a post here: https://medium.com/@gautam.e/hello-mojo-83f5a8b04074.
